pkg/evaluator/builtin.go

```diff
@@ -155,16 +166,24 @@ func (rt *UnimplementedRuntime) Unimplemented(s string) {
 	rt.Print(fmt.Sprintf("%q not implemented\n", s))
 }
 
-func (rt *UnimplementedRuntime) Read() string          { rt.Unimplemented("read"); return "" }
-func (rt *UnimplementedRuntime) Sleep(_ time.Duration) { rt.Unimplemented("sleep") }
-func (rt *UnimplementedRuntime) Yielder() Yielder      { rt.Unimplemented("yielder"); return nil }
-func (rt *UnimplementedRuntime) Move(x, y float64)     { rt.Unimplemented("move") }
-func (rt *UnimplementedRuntime) Line(x, y float64)     { rt.Unimplemented("line") }
-func (rt *UnimplementedRuntime) Rect(x, y float64)     { rt.Unimplemented("rect") }
-func (rt *UnimplementedRuntime) Circle(r float64)      { rt.Unimplemented("circle") }
-func (rt *UnimplementedRuntime) Width(w float64)       { rt.Unimplemented("width") }
-func (rt *UnimplementedRuntime) Color(s string)        { rt.Unimplemented("color") }
-func (rt *UnimplementedRuntime) Clear(color string)    { rt.Unimplemented("clear") }
+func (rt *UnimplementedRuntime) Read() string            { rt.Unimplemented("read"); return "" }
+func (rt *UnimplementedRuntime) Sleep(_ time.Duration)   { rt.Unimplemented("sleep") }
+func (rt *UnimplementedRuntime) Yielder() Yielder        { rt.Unimplemented("yielder"); return nil }
+func (rt *UnimplementedRuntime) Move(x, y float64)       { rt.Unimplemented("move") }
+func (rt *UnimplementedRuntime) Line(x, y float64)       { rt.Unimplemented("line") }
+func (rt *UnimplementedRuntime) Rect(x, y float64)       { rt.Unimplemented("rect") }
+func (rt *UnimplementedRuntime) Circle(r float64)        { rt.Unimplemented("circle") }
+func (rt *UnimplementedRuntime) Width(w float64)         { rt.Unimplemented("width") }
+func (rt *UnimplementedRuntime) Color(s string)          { rt.Unimplemented("color") }
+func (rt *UnimplementedRuntime) Clear(color string)      { rt.Unimplemented("clear") }
+func (rt *UnimplementedRuntime) Stroke(s string)         { rt.Unimplemented("stroke") }
+func (rt *UnimplementedRuntime) Fill(s string)           { rt.Unimplemented("fill") }
+func (rt *UnimplementedRuntime) Text(s string)           { rt.Unimplemented("text") }
```

Comment:
I think you've accidentally snuck in some bits of the next PR (i.e. Text et al are not implemented yet, although technically thats exactly what this function says).

Reply:
fixed, thanks.